cms/djangoapps/contentstore/views/tests/test_assets.py

```diff
@@ -86,6 +88,45 @@ def test_pdf_asset(self):
         # Note: Actual contentType for textbook.pdf in asset.json is 'text/pdf'
         self.assertEqual(content.content_type, 'application/pdf')
 
+    def test_relative_url_for_split_course(self):
+        """
+        Test relative path for split courses assets
+        """
+        with modulestore().default_store(ModuleStoreEnum.Type.split):
+            module_store = modulestore()
+            course_id = module_store.make_course_key('edX', 'toy', '2012_Fall')
+            import_course_from_xml(
+                module_store,
+                self.user.id,
+                TEST_DATA_DIR,
+                ['toy'],
+                static_content_store=contentstore(),
+                target_id=course_id,
+                create_if_not_present=True
+            )
+            course = module_store.get_course(course_id)
+
+            filename = 'sample_static.txt'
+            html_src_attribute = '"/static/{filename}"'.format(filename=filename)
```

Comment:
Nit: you can also just do `"/static/{}".format(filename)`, which is less-wordy and more readable.